dev-tools/mage/crossbuild.go

```diff
@@ -271,6 +271,13 @@ func (b GolangCrossBuilder) Build() error {
 		return errors.Wrap(err, "failed to determine mage-linux-"+builderArch+" relative path")
 	}
 
+	// If we're running this mage target on Windows, the path separator used in
+	// buildCmd will be `\`. But the buildCmd is executed inside a Linux Docker
+	// container. So we replace all `\` path separators with `/`.
+	if GOOS == "windows" {
+		buildCmd = strings.ReplaceAll(buildCmd, "\\", "/")
```

Comment:
would you consider using  [filepath.ToSlash](https://pkg.go.dev/path/filepath#ToSlash) ?   I think if we do that we don't have to have the `GOOS == "windows"` check.  If the `os.PathSeparator` is `/` it won't do anything, otherwise it will replace all instances of `os.PathSeparator` with `/`.

Reply:
I like it, thanks for the suggestion @leehinman. Changed in 39daec4db.

Reply:
I like it, thanks for the suggestion @leehinman. Changed in 39daec4db.